baseplate/lib/thrift_pool.py

```diff
@@ -165,7 +168,11 @@ def __init__(
         self.protocol_factory = protocol_factory
 
         self.size = size
-        self.pool: ProtocolPool = queue.LifoQueue()
+        if config.queue == "fifo":
```

Comment:
`config.queue` doesn't exist so I think it wants to be passed in through init's kwargs